gen/calc.py

```diff
@@ -371,6 +371,7 @@ def validate_os_type(os_type):
         'ui_banner_footer_content': 'null',
         'ui_banner_image_path': 'null',
         'ui_banner_dismissible': 'null',
+        'dcos_overlay_config_attempts': '4',
```

Comment:
Validation that the provided string by the user is an integer and between 0 and 1000 I think would be good.